packages/middleware-retry/package.json

```diff
@@ -14,12 +14,13 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "@aws-sdk/protocol-http": "1.0.0-gamma.1",
     "@aws-sdk/service-error-classification": "1.0.0-gamma.1",
     "@aws-sdk/types": "1.0.0-gamma.1",
-    "tslib": "^1.8.0"
+    "tslib": "^1.8.0",
+    "uuid": "^8.0.0"
```

Comment:
Just fyi, uuid doesn't work in RN directly. See here: https://github.com/uuidjs/uuid#react-native

Reply:
Refs: [#1216/commits/18945e17b663ec3e0b52187521017709576273ad](https://github.com/aws/aws-sdk-js-v3/pull/1216/commits/18945e17b663ec3e0b52187521017709576273ad)

Reply:
Updated in [commit](https://github.com/aws/aws-sdk-js-v3/pull/1266/commits/c29719146b02577475624ca0e810decde69d46fd)